cfn-resources/alert-configuration/cmd/resource/resource.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//      http://www.apache.org/licenses/LICENSE-2.0
+//         http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
was this intended?

Reply:
fixed, was my Copyright settings in IntelliJ